### PR TITLE
Add tv show view

### PR DESCRIPTION
**Pull request by grafixeyehero**
**Changes**
* Add tv show view
* Extract reusable component to reduce duplication code

### CODE REVIEW

File: src/apps/experimental/routes/shows/UpComingView.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react';
+import Box from '@mui/material/Box';
+import { useGetGroupsUpcomingEpisodes } from 'hooks/useFetchItems';
+import Loading from 'components/loading/LoadingComponent';
+import globalize from 'scripts/globalize';
+import SectionContainer from '../../components/library/SectionContainer';
+import { LibraryViewProps } from 'types/library';
+
+const UpComingView: FC<LibraryViewProps> = ({ parentId }) => {
```
Comment (thornbill) [inline src/apps/experimental/routes/shows/UpComingView.tsx:9]:
```suggestion
const UpcomingView: FC<LibraryViewProps> = ({ parentId }) => {
```

File: src/hooks/useFetchItems.ts [hunk 2]
```diff
@@ -509,3 +513,91 @@ export const useGetItemsViewByType = (
             ].includes(viewType) && !!parentId
     });
 };
+
+type GroupsUpcomingEpisodes = {
+    name: string;
+    items: BaseItemDto[];
+};
+
+function gropsUpcomingEpisodes(items: BaseItemDto[]) {
```
Comment (thornbill) [inline src/hooks/useFetchItems.ts:522]:
```suggestion
function groupsUpcomingEpisodes(items: BaseItemDto[]) {
```

**Review by thornbill — CHANGES_REQUESTED** [review]

**Comment by grafixeyehero** [comment]
@thornbill sorry for I force-pushed the commits. 😥

File: src/hooks/useFetchItems.ts (hunk 2 above)
Comment (grafixeyehero) [reply src/hooks/useFetchItems.ts]: my bad!

**Review by thornbill — APPROVED** [review]

**Pull request merged**